contracts/Joint.sol

```diff
@@ -872,13 +858,12 @@ abstract contract Joint {
         address[] memory swapPath,
         uint256 swapInAmount,
         uint256 minOutAmount
-    ) external onlyGovernance returns (uint256) {}
+    ) external onlyGovernance virtual returns (uint256) {}
```

Comment:
warning: if you override a function with a modifier, do you keep the modifier?

if you don't, we should remove it from here. 

also, as this has to be overriden, I wouldn't add {} but ; at the end of line, so the compiler throws if not implemented.

Reply:
Right - implementing